web/js/layers/layers.model.js

```diff
@@ -182,7 +181,7 @@ wv.layers.model = wv.layers.model || function (models, config) {
   };
 
   self.remove = function (id) {
-    var index = _.findIndex(self.active, {
+    var index = findIndex(self.active, {
```

Comment:
I still feel that it would be useful to use `_findIndex` just so someone doesn't have to click around and search for it until they find it in the list of imports. I would assume there is a local findIndex function if I saw this.

Reply:
There *is* a local `findIndex`, if you're importing it. An `_` before a variable name typically means it's a private variable, and that could be confusing as well.

Reply:
That's a good point. Maybe another symbol in that case. When I see `_.something` I know to look it up in the lodash docs to see exactly what parameters it takes and whatnot. can save time.

Reply:
We could do something like `import { findIndex as lodashFindIndex } from 'lodash'`, but I don't think that's necessary.